client/js/widgets/widget.js

```diff
@@ -1967,6 +1967,8 @@ export class Widget extends StateManaged {
     if (limit.maxY !== undefined) {
       newCoord.y = Math.min(limit.maxY, newCoord.y);
     }
+    newCoord.x = Math.round(newCoord.x);
+    newCoord.y = Math.round(newCoord.y);
```

Comment:
You should do the rounding in the dragCorner method so that it's used for all 3 locations which determine drag location.

Reply:
Is the commit I just made what you are talking about?  The only change I noticed in testing is that now the CSS Transform tutorial locations on the table are integers.  The scale test and simple cases seem the same.